.circleci/config.yml

```diff
@@ -14,3 +14,12 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-app-catalog:
+          name: "package and push nginx-ingress-controller-app chart"
+          app_catalog: "default-catalog"
+          app_catalog_test: "default-test-catalog"
+          chart: "nginx-ingress-controller-app"
+          # Trigger job on git tag.
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
Since these are separate repos it should be "thread safe" to push to both catalogs at once.

For info for Aqua Julien needed to add a requires block but this was to push 3 charts to the same catalog.

https://github.com/giantswarm/aqua-app/blob/0497576d10678ed24587bc339432eb6852113ed2/.circleci/config.yml#L26-L27

Reply:
I think with the separate catalogs there should be no issues. 
But good to know about this requirements for multiple charts. :+1: